tests/expectations.json

```diff
@@ -32179,28 +32143,16 @@
   "language/statements/for-of/int8array-mutate.js": "CRASH",
   "language/statements/for-of/int8array.js": "CRASH",
   "language/statements/for-of/iterator-as-proxy.js": "CRASH",
-  "language/statements/for-of/iterator-close-non-object.js": "CRASH",
-  "language/statements/for-of/iterator-close-non-throw-get-method-abrupt.js": "CRASH",
   "language/statements/for-of/iterator-close-non-throw-get-method-is-null.js": "CRASH",
-  "language/statements/for-of/iterator-close-non-throw-get-method-non-callable.js": "CRASH",
-  "language/statements/for-of/iterator-close-throw-get-method-abrupt.js": "CRASH",
-  "language/statements/for-of/iterator-close-throw-get-method-non-callable.js": "CRASH",
-  "language/statements/for-of/iterator-close-via-break.js": "CRASH",
   "language/statements/for-of/iterator-close-via-continue.js": "CRASH",
   "language/statements/for-of/iterator-close-via-return.js": "CRASH",
   "language/statements/for-of/iterator-close-via-throw.js": "CRASH",
-  "language/statements/for-of/iterator-next-error.js": "CRASH",
-  "language/statements/for-of/iterator-next-reference.js": "CRASH",
-  "language/statements/for-of/iterator-next-result-done-attr.js": "CRASH",
   "language/statements/for-of/iterator-next-result-type.js": "CRASH",
-  "language/statements/for-of/iterator-next-result-value-attr-error.js": "CRASH",
-  "language/statements/for-of/iterator-next-result-value-attr.js": "CRASH",
+  "language/statements/for-of/iterator-next-result-value-attr-error.js": "TIMEOUT",
```

Comment:
**nitpick:** Would be better to add this test to `skip.json`.

Reply:
I thought I'd leave this in: The reason it's failing is because we don't support getters and setters yet, so an iterator never breaks.

That will be fixed very shortly, and it's also a properly valid "timeout" case instead of just being "copying 4GB of memory isn't fast enough" :)

Reply:
Fair enough.